cms/templates/js/move-xblock-list.underscore

```diff
@@ -27,7 +33,7 @@
                         </span>
                     <% } %>
                     <span class="icon fa fa-arrow-right forward-sr-icon" aria-hidden="true"></span>
-                    <span class="sr forward-sr-text"><%- forwardButtonSRText %></span>
+                    <span class="sr forward-sr-text"><%- gettext("Click for children") %></span>
```

Comment:
@muhammad-ammar is this the sr text for which @andy-armstrong suggested "View child components"? I think "View child components" is more clear than "Click for children".